src/Microsoft.DotNet.Interactive.HttpRequest.Tests/ParserTests.cs

```diff
@@ -115,13 +133,13 @@ public void newline_is_legal_at_the_end_of_a_request()
         {
             var result = HttpRequestParser.Parse(
                 """
-        GET https://example.com
+                GET https://example.com
 
-        """);
+                """);
 
             result.SyntaxTree.RootNode
                   .ChildNodes.Should().ContainSingle<HttpRequestNode>().Which
-                  .UrlNode.ChildTokens.Last().TextWithTrivia.Should().Be("\r\n");
+                  .UrlNode.ChildTokens.Last().TextWithTrivia.Should().Contain("\r\n");
```

Comment:
Same here - it would be betterr to assert that there are 3 tokens with token.Kind is TokenKind.Newline instead. That would eliminate the need to check the exact text here - exact text for newline tokens is already validated by the lexer test for newlines above.

Reply:
Also fixed.